src/routes/CategoriesRoutes.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+const CategoriesRouter = express.Router();
+
+//Get a list of categories 
+CategoriesRouter.get('/', (req, res) => {
+    res.json({
+        message: 'hello categories'
+    });
+});
+
+//Create a new category
+CategoriesRouter.post('/', (req, res) => {
+
+});
+
+//Update category by slug
```

Comment:
I have not used slugs before, but have you done your reseach on how to use them. I don't think this is how you use them.  If you're not sure, we can discuss tomorrow. 

But for now, just don't worry about the slug, just create the routes as how we have learnt. 

From a quick research,